packages/@webex/plugin-meetings/test/unit/spec/members/index.js

```diff
@@ -399,30 +435,107 @@ describe('plugin-meetings', () => {
         await checkInvalid(
           resultPromise,
           'The associated locus url for this meetings members object must be defined.',
-          spies
+          spies,
+        );
+      });
+
+      it('should not make a request if locus throws ReclaimHostNotSupportedError', async () => {
```

Comment:
this test and the other new ones have wrong titles - they all say "should not make a request", while in fact assignRoles() does make the request in all of them. They should probably instead say "throws XXX" where XXX is the right SDK error

Reply:
actually the second part of these test titles is also incorrect - it should say "if locus returns ZZZ code" where ZZZ is the right code